### ISSUE TITLE
This extension is kind of useless. Use `EXT:schema` instead.

### ISSUE THREAD

**Issue by simonschaufi** [hyperdigital/hd_structureddata#1]
Or use schema as dependency and their api to extend it with custom code but don't reinvent the wheel :)

**Comment by martinpribylhd** [comment]
Hi, thank you for the suggestion. I tried schema extension in just a few examples, I’ll try it more deeply. 

The reason of this ext was more about to make it more easier work for one our customer. They wanted to have backed options to add structured data for their products and news. And each entry has totally different options.

**Comment by mweber1973** [comment]
> Or use schema as dependency and their api to extend it with custom code but don't reinvent the wheel :)

I think LD+JSON ist the future, because its easier to read for KI-robots or search engines.....

**Comment by xmental** [comment]
I completely agree with you—it’s a really convenient feature for customers, especially when it comes to independently and flexibly tagging products, events, or news with structured data. Please keep it up.
